### PR TITLE
First Capstone Project

### PR DESCRIPTION
**Pull request by Sajeelzafar**
## In this project the following things have been designed:

- Home Page of the travel and tour website with personalized content.
- About Page of travels and tour website with personalized content.
- The format is kept similar to the one provided in the capstone project.
- Hovering effect is created on the navbar, mobile menu, and buttons.
- The buttons created are made interactive using JavaScript.
- The live video link to the presentation of the website is attached in the README file along with the live website link.

### CODE REVIEW

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @Sajeelzafar  

Great work on making Progress and design is beautiful 👍 👏 !
You've done well implementing project requirement, but there are still some which aren't addressed yet.

## Highlights

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README 

almost

## Required Changes ♻️
_Check the comments below the review._

- [x] Great job documenting your project! However,  please added  an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) In your README Files.

- [x] For HTML/CSS & JavaScript capstone project, you required to use the linters configuration of [html&css + javascript](https://github.com/microverseinc/linters-config/blob/master/html-css-js) (for projects that require you to use both JavaScript and HTML & CSS), But You used html&css linters.yml file kindly update with [linters.yml](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml)
- [x] For the section "Featured speakers (In the case of yours "Team" section) should be created dynamically in JavaScript. You are Required to use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically. Kindly. Work on it.
https://github.com/Sajeelzafar/Conference_Page/blob/5526f447182e27f863c220e8319123451f20e197/index.html#L124-L230

- [x] Kindly Adjust navigation between the pages, to  link from About page to Home page For Desktop screen size.
https://github.com/Sajeelzafar/Conference_Page/blob/5526f447182e27f863c220e8319123451f20e197/about.html#L32-L46


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Sajeelzafar** [comment]
Since "DOMContentLoaded" is used as an event listener to load the team using JavaScript, kindly refresh the page when switching from mobile view to desktop view or vice versa for optimum performance.

File: README.md [hunk 1]
```diff
@@ -57,9 +59,10 @@ Feel free to check the [issues page](../../issues/).
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
-
+- Cindy Shinon Behance for her UI design template
```
Comment (cynthiainga) [inline README.md]: - [ ] As mentioned by the previous reviewer you should add an "Acknowledgements" section or similar that gives appropriate credit to[Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), not mention it he but also add the links 🙃 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Sajeelzafar, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Used correct Github flow ✔️  

 ## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — APPROVED** [review]
Hi  @Sajeelzafar , 

Good work implementing the changes requested. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)


## Highlights

✅ No linter error
✅ README is okay
✅ Good Design 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**